There are some test failures and I do not think my changes are relevant.

The test `Test - Unit - Template / Unit - Generate test matrix (push)` is failing due to

```
Prepare all required actions
Run ./.github/actions/test-type-vars
Run matrix_file="charts//test/ci-test-config.yaml"
CHART_PATH=charts/
Error: open charts//test/ci-test-config.yaml: no such file or directory
Error: Process completed with exit code 1.
```

I don't know if the file `ci-test-config.yaml` is indeed where this CI job expects it to be.

The test `Test - Chart Version / 8.10 - nosec - install - pr - gke / gke - ITs / nosec - install - gke / install for install on gke - nosec (pull_request)` is faiing because the zeebe pods are not starting.

The `configmap/integration-zeebe-configuration` contains `camunda.data.secondary-storage.elasticsearch.url: "https://elasticsearch-21-6-3.ci.distro.ultrawombat.com:443"` and this does not seem to resolve to a known address from within the pods.

--- a/charts/camunda-platform-8.10/templates/common/_helpers.tpl
+++ b/charts/camunda-platform-8.10/templates/common/_helpers.tpl
@@ -477,7 +477,7 @@ Orchestration Identity templates.
 [camunda-platform] Orchestration Identity external URL.
 */}}
 {{- define "camundaPlatform.orchestrationIdentityExternalURL" }}
-  {{- printf "%s/identity" (include "camundaPlatform.orchestrationExternalURL" . | trimSuffix "/") -}}
+  {{- printf "%s/admin" (include "camundaPlatform.orchestrationExternalURL" . | trimSuffix "/") -}}
 {{- end -}}
 
 
@@ -759,7 +759,7 @@ Release templates.
     url: {{ include "camundaPlatform.tasklistExternalURL" . }}
     readiness: {{ printf "%s%s" $baseURLInternal (include "camundaPlatform.joinpath" (list .Values.orchestration.contextPath .Values.orchestration.readinessProbe.probePath)) }}
     metrics: {{ printf "%s%s" $baseURLInternal (include "camundaPlatform.joinpath" (list .Values.orchestration.contextPath .Values.orchestration.metrics.prometheus)) }}
-  - name: Orchestration Identity
+  - name: Orchestration Admin
     id: orchestrationIdentity
     version: {{ include "camundaPlatform.imageTagByParams" (dict "base" .Values.global "overlay" .Values.orchestration) }}
     url: {{ include "camundaPlatform.orchestrationIdentityExternalURL" . }}

--- a/charts/camunda-platform-8.10/templates/common/_helpers.tpl
+++ b/charts/camunda-platform-8.10/templates/common/_helpers.tpl
@@ -474,7 +474,7 @@ Orchestration Identity templates.
 */}}
 
 {{/*
-[camunda-platform] Orchestration Identity external URL.
+[camunda-platform] Orchestration Admin external URL.
 */}}
 {{- define "camundaPlatform.orchestrationIdentityExternalURL" }}
   {{- printf "%s/admin" (include "camundaPlatform.orchestrationExternalURL" . | trimSuffix "/") -}}

--- a/charts/camunda-platform-8.10/test/unit/console/golden/configmap.golden.yaml
+++ b/charts/camunda-platform-8.10/test/unit/console/golden/configmap.golden.yaml
@@ -61,9 +61,9 @@ data:
               url: http://localhost:8080/tasklist
               readiness: http://camunda-platform-test-zeebe.camunda-platform:9600/actuator/health/readiness
               metrics: http://camunda-platform-test-zeebe.camunda-platform:9600/actuator/prometheus
-            - name: Orchestration Identity
+            - name: Orchestration Admin
               id: orchestrationIdentity
-              url: http://localhost:8080/identity
+              url: http://localhost:8080/admin
               readiness: http://camunda-platform-test-zeebe.camunda-platform:9600/actuator/health/readiness
               metrics: http://camunda-platform-test-zeebe.camunda-platform:9600/actuator/prometheus
           

